docs/pages/push-notifications/fcm-credentials.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Add Google Service Account Keys using FCM V1'
+sidebar_title: Add Android FCM V1 credentials
+description: Learn how to create a Google Service Account Key for sending Android Notifications using FCM V1
+---
+
+import { Step } from '~/ui/components/Step';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Tab, Tabs } from '~/ui/components/Tabs';
+
+## Create a new Google Service Account Key
+
+Here are the steps to configure a new Google Service Account Key in EAS for sending Android Notifications using FCM V1.
+
+<Step label="1">
+  Create a new Firebase project for your app in the [Firebase Console](https://console.firebase.google.com). If you already have a Firebase project for your app, continue to the next step.
+
+  <ImageSpotlight
+    alt="Create a project in the Firebase Console"
+    src="/static/images/creating-google-service-account/fcm-v1/new-service-account/01-new-firebase-project.png"
+    style={{ maxWidth: 800 }}
+  />
+</Step>
+
+<Step label="2">
+  In the Firebase console, open Settings > [Service Accounts](https://console.firebase.google.com/project/_/settings/serviceaccounts/adminsdk) for your project.
```

Comment:
```suggestion
  In the Firebase console, open **Project settings** > [**Service accounts**](https://console.firebase.google.com/project/_/settings/serviceaccounts/adminsdk) for your project.
```

- Let's highlight the settings name by making them bold (this is a convention we follow)
- Let's also use the same settings names as Firebase does (based on the screenshot)